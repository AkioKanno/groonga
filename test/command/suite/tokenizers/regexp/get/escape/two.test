table_create Lexicon TABLE_PAT_KEY ShortText \
  --default_tokenizer TokenRegexp
table_tokenize Lexicon "c:\\server" --mode ADD

table_tokenize Lexicon "c:\\\\server" --mode GET
