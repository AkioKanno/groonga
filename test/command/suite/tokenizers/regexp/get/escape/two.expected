table_create Lexicon TABLE_PAT_KEY ShortText   --default_tokenizer TokenRegexp
[[0,0.0,0.0],true]
table_tokenize Lexicon "c:\\server" --mode ADD
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "￯",
      "position": 0
    },
    {
      "value": "c:",
      "position": 1
    },
    {
      "value": ":\\",
      "position": 2
    },
    {
      "value": "\\s",
      "position": 3
    },
    {
      "value": "se",
      "position": 4
    },
    {
      "value": "er",
      "position": 5
    },
    {
      "value": "rv",
      "position": 6
    },
    {
      "value": "ve",
      "position": 7
    },
    {
      "value": "er",
      "position": 8
    },
    {
      "value": "r",
      "position": 9
    },
    {
      "value": "￰",
      "position": 10
    }
  ]
]
table_tokenize Lexicon "c:\\\\server" --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "c:",
      "position": 0
    },
    {
      "value": "\\s",
      "position": 2
    },
    {
      "value": "er",
      "position": 4
    },
    {
      "value": "ve",
      "position": 6
    },
    {
      "value": "er",
      "position": 7
    }
  ]
]
